scalafix-core/src/main/scala/scalafix/rewrite/ExplicitImplicit.scala

```diff
@@ -1,21 +1,20 @@
-package scalafix.rewrite
+package scalafix
+package rewrite
 
-import scala.{meta => m}
-import scalafix.util.Patch
+import scala.meta._
 import scalafix.util.Whitespace
 import scala.collection.immutable.Seq
-import scalafix.util.TokenPatch
 
-case object ExplicitImplicit extends Rewrite[ScalafixMirror] {
+case class ExplicitImplicit(mirror: Mirror) extends SemanticRewrite(mirror) {
```

Comment:
It used to be a parameter of `rewrite`. One of the major motivations for this PR was to remove `mirror` from the `rewrite`. The short answer why is that type erasure makes it hard to do this safely, the long explanation is here f18bd69a5ffaf083a9a580114c9b09b074f19cf6